MeshGeoToolsLib/GeoMapper.cpp

```diff
@@ -110,17 +110,6 @@ void GeoMapper::mapOnMesh(MeshLib::Mesh const*const mesh)
     delete _grid;
 }
 
-void GeoMapper::mapToConstantValue(double value)
```

Comment:
Leave this here please.

Reply:
No problem. Dropped the commit. A test for that function would be nice, otherwise it appears on the next cleanup session again.